pom.xml

```diff
@@ -1472,6 +1474,46 @@
         <groupId>org.scalatest</groupId>
         <artifactId>scalatest-maven-plugin</artifactId>
       </plugin>
+      <!-- Build the JARs-->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>${maven-jar-plugin.version}</version>
+        <configuration>
+          <!-- Configuration of the archiver -->
+          <archive>
```

Comment:
Is this config needed? I haven't usually needed anything more than the executions below in this kind of incantation.